features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/OpenAPI/ApiCreateOpenAPI.jsx

```diff
@@ -240,19 +257,6 @@ export default function ApiCreateOpenAPI(props) {
                                 </Button>
```

Comment:
Shall we add permission checking condition to this button and other places too?

Reply:
We won't add this to here as the logic comes with isCreating prop